tools/snippet_shell.py

```diff
@@ -45,24 +45,24 @@ def _start_services(self, console_env):
         console_env['s'] = self._ad.snippet
 
     def _get_banner(self, serial):
-        return """
-Connected to {}. Call methods against:
-    ad (android_device.AndroidDevice)
-    snippet or s (Snippet)
-    """.format(serial)
-
+        lines = ['Connected to %s.' % serial,
+                 'Call methods against:',
+                 '    ad (android_device.AndroidDevice)',
+                 '    snippet or s (Snippet)']
+        return '\n'.join(lines)
```

Comment:
Is this really clearer to read? What was wrong with the old way? Also the new line breaks are different; is that intentional?